proof/crefine/ARM_HYP/ArchMove_C.thy

```diff
@@ -634,6 +634,38 @@ crunches insertNewCap, Arch_createNewCaps, threadSet, Arch.createObject, setThre
    simp: unless_def updateObject_default_def crunch_simps
    ignore_del: preemptionPoint)
 
+(* FIXME MK: consider move *)
+lemma addrFromPPtr_mask[simplified ARM_HYP.pageBitsForSize_simps]:
+  "n \<le> pageBitsForSize ARMSuperSection
+   \<Longrightarrow> addrFromPPtr ptr && mask n = ptr && mask n"
```

Comment:
Agreed, this and the next one should probably go relatively high up in AInvs somewhere. Maybe ArchInvariants_AI.